testing/skia_gold_client/lib/skia_gold_client.dart

```diff
@@ -319,21 +358,6 @@ class SkiaGoldClient {
     // image size because those wrong pixels are constrained by
     // `pixelDeltaThreshold` below.
     final int maxDifferentPixels = (screenshotSize * kMaxDifferentPixelsRate).toInt();
```

Comment:
You may want to parametrize this one too. It controls how many pixels in the new image are allowed to differ from the golden image.

Right now, this is hardcoded at 10% but you may want to set a different value on other platforms.

Reply:
👍 done